### PR TITLE
Updated conditions for third party report

### PR DESCRIPTION
**Pull request by dhairyya**
Updated documentation to reduce escalation for third party reporting

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda and @MSFTTracyP could you please approve this?

File: microsoft-365/security/office-365-security/user-submission.md [hunk 1]
```diff
@@ -194,6 +194,16 @@ To correctly identify the original attached messages, messages sent to the custo
 To specify the reason why the original attached messages were reported, messages sent to the user submissions mailbox must meet the following criteria:
 
 - The original message attachment is unmodified.
+- The reported message should contains below required headers
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
- The reported message should contain the following required headers:
```

File: microsoft-365/security/office-365-security/user-submission.md [hunk 2]
```diff
@@ -194,6 +194,16 @@ To correctly identify the original attached messages, messages sent to the custo
...
+  -  1.X-Microsoft-Antispam-Message-Info
+  -  2. Message-Id
+  -  3. X-Ms-Exchange-Organization-Network-Message-Id
+  -  4. X-Ms-Exchange-Crosstenant-Id
+
+> [!INFO]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**